frontend/src/components/module-editor/SideBar.tsx

```diff
@@ -98,113 +154,139 @@ const Sidebar = (): React.ReactElement => {
+                  </Text>
+                </Flex>
+              </Box>
+              <Tabs variant="unstyled" height="100%" overflowY="hidden">
+                <Box
+                  bg="background.light"
+                  borderRadius="md"
+                  p="6px"
+                  margin="auto"
+                  maxW="min-content"
+                  my="43px"
+                >
+                  <TabList>
+                    <Tab
+                      style={{ paddingInline: "35px", height: "32px" }}
+                      _selected={{
+                        color: "white",
+                        bg: "brand.royal",
+                        borderRadius: "md",
+                      }}
+                    >
+                      Overview
+                    </Tab>
+                    <Tab
+                      style={{ paddingInline: "23px", height: "32px" }}
+                      _selected={{
+                        color: "white",
+                        bg: "brand.royal",
+                        borderRadius: "md",
+                      }}
+                    >
+                      Components
+                    </Tab>
+                  </TabList>
+                </Box>
+                <TabPanels height="100%" overflowY="hidden">
+                  <TabPanel
+                    p="0"
+                    pb="200px"
+                    height="100%"
+                    overflowY="auto"
+                    className="tabScroll"
+                  >
+                    <ModuleOverview />
+                  </TabPanel>
+                  <TabPanel
+                    pb="200px"
+                    height="100%"
+                    overflowY="auto"
+                    className="tabScroll"
+                  >
+                    <ContentKiosk />
+                  </TabPanel>
+                </TabPanels>
+              </Tabs>
+              <Spacer />
+              {Object.values(state.hasChanged).length ? (
+                <Button
+                  bg="#5FCA89"
+                  color="white"
+                  leftIcon={<SaveIcon />}
+                  borderRadius="0"
+                  pl="35px"
+                  width="100%"
+                  h="55px"
+                  justifyContent="left"
+                  onClick={() => saveChanges(state.hasChanged)}
+                >
+                  Save Changes
+                </Button>
+              ) : (
+                <></>
+              )}
+            </Flex>
+            {isOpen && (
+              <EditModuleModal
+                module={
+                  courseData.course.modules[
+                    Number(moduleIndex)
+                  ] as ModuleResponse
+                }
+                isOpen={isOpen}
+                onClose={onClose}
+                formatCourseRequest={formatCourseRequest}
+              />
+            )}
+          </Box>
         )}
-      </Flex>
-    </Box>
+      ;
```

Comment:
Also forgot to mention, is this a typo? I believe a `;` in JSX just renders as a literal `;`.

Reply:
Yes, good catch!